src/wp-admin/css/list-tables.css

```diff
@@ -1580,18 +1580,6 @@ div.action-links,
 	margin-top: 0;
 }
 
-.plugin-card .column-description .authors {
-	order: 1;
-}
-
-.plugin-card .column-description .plugin-dependencies {
-	order: 2;
-}
-
-.plugin-card .column-description p:empty {
-	display: none;
-}
```

Comment:
I guess the description can still be empty? `<p><?php echo $description; ?></p>` 
In that case, it would be nice to either not print out the empty paragraph or keep hiding it via CSS.

Reply:
Added back in [4dbee95](https://github.com/WordPress/wordpress-develop/pull/6078/commits/4dbee959f47ff0c5ded7e6f26f219b26398c2688).